contrib/dockerd-rootless.sh

```diff
@@ -54,6 +54,21 @@ if ! [ -d "$HOME" ]; then
 	exit 1
 fi
 
+mount_directory() {
+	if [ -z "$_DOCKERD_ROOTLESS_CHILD" ]; then
+		echo "mount_directory should be called from the child context. Otherwise data loss is at risk"
+		exit 1
+	fi
+
+	local directory="$1"
+	local mount_options="${2:---bind}"
+
+	directory_realpath=$(realpath "$directory")
+	rm -rf "$directory"
```

Comment:
A comment should be added to explain that this is a "copied-up" file (usually symlink)

Reply:
Updated.